pallets/stableswap/src/types.rs

```diff
@@ -0,0 +1,98 @@
+use sp_runtime::Permill;
+use sp_std::vec;
+use sp_std::vec::Vec;
+
+use codec::{Decode, Encode, MaxEncodedLen};
+use scale_info::TypeInfo;
+use sp_core::RuntimeDebug;
+use sp_runtime::traits::Zero;
+
+pub(crate) type Balance = u128;
+
+/// Pool identifier. Share Asset id is used as pool identifier.
+/// Share asset is unique token for each pool. That means using share asset as pool identifier
+/// does not require additional "tracking" id for newly created pools.
+#[derive(Clone, Copy, PartialEq, Encode, Decode, RuntimeDebug, MaxEncodedLen, TypeInfo)]
```

Comment:
```suggestion
#[derive(Clone, Copy, PartialEq, Encode, Decode, RuntimeDebug, MaxEncodedLen, TypeInfo, Eq)]
```
This type will be used here: https://github.com/galacticcouncil/warehouse/blob/81fa8ae88fa24139a2ee9c88eeba5042c5fd435d/liquidity-mining/src/lib.rs#L164